hikari/impl/entity_factory.py

```diff
@@ -3928,3 +3939,44 @@ def deserialize_sku(self, payload: data_binding.JSONObject) -> monetization_mode
             slug=payload["slug"],
             flags=monetization_models.SKUFlags(payload["flags"]),
         )
+
+    ###############
+    # POLL MODELS #
+    ###############
+
+    def _deserialize_poll_media(self, payload: data_binding.JSONObject) -> poll_models.PollMedia:
+        return poll_models.PollMedia(
+            text=payload.get("text", None),
```

Comment:
```suggestion
            text=payload.get("text"),
```

The default is already None